### DISCUSSION TITLE
Add a prop which allows you to set Menu to be normally open

### DISCUSSION THREAD

**Discussion by marijnyoast** [tailwindlabs/headlessui#336] [Ideas]
In our designs we have collapsible Menu's in our sidebar that are initially open, but can be closed by the user. This behaviour is currently impossible with the Menu component since the `open` state is managed internally and we can't alter that.

So we would really like an additional prop that allows you to tell the `Menu` or `Menu.Items` that it needs to be initially open.

**Comment by bezreyhan** [comment]
I think this would also be related to this discussion I started here: https://github.com/tailwindlabs/headlessui/discussions/335

**Comment by adamfortuna** [comment]
I ran into this while trying to open a menu programmatically and noticed there wasn't an API for that. My use case is similar – I'm showing a Menu on a landing page and want to show it expanded initially to present it to the user. In this case it would start open, but then the user could interact with it and close it. 

On the one hand I can see why it's less important. If the user clicks anywhere on the page it'd close all expanded buttons.
